inference-engine/tests/functional/inference_engine/ngraph_reader/relu_tests.cpp

```diff
@@ -220,8 +220,4 @@ TEST_F(NGraphReaderTests, ReadReLUScalarNetwork) {
     Blob::CPtr blob;
     Core reader;
     auto nGraph = reader.ReadNetwork(model, blob);
-    CNNNetwork cnetwork(nGraph);
-    IE_SUPPRESS_DEPRECATED_START
-    cnetwork.begin();
```

Comment:
I think this test should validate that conversion works without exceptions. But conversion doesn't happen if you don't call it.

Reply:
reverted back